torch/package/find_file_dependencies.py

```diff
@@ -11,15 +11,19 @@ class _ExtractModuleReferences(ast.NodeVisitor):
     """
 
     @classmethod
-    def run(cls, src: str, package: str) -> List[Tuple[str, Optional[str]]]:
-        visitor = cls(package)
+    def run(
+        cls, src: str, package: str, module: str, failed_dunder_imports: List[str]
```

Comment:
I would ordinarily be suspicious of a change like this given that this is a `@classmethod`, but [it does not seem to be one defined in `ast.NodeVisitor`](https://github.com/python/cpython/blob/3.8/Lib/ast.py#L347), and `_ExtractModuleReferences` does not have any subclasses. I see also that `find_files_source_depends_on` is set to  `_ExtractModuleReferences.run` at the end of the file, which also seems weird to me.

Reply:
Yeah, I think the setup of this file is interesting. It reads to me like Zach was possibly trying to hide the implementation of the class completely with the use of `find_files_source_depends_on`.